### DISCUSSION TITLE
对JUNO第二阶段大作业平台说明的若干问题

### DISCUSSION THREAD

**Discussion by mhliu0001** [physics-data/faq#164] [Q&A] [answered]
1. PEType的说明问题。我知道PEType=0对应契伦科夫辐射，但似乎没有文档提到过这一点。[平台的说明](https://physics-data.meow.plus/challenges/pd2021-juno)中只提到-1对应暗噪声，没有提到0对应什么。
2. 训练集数量问题。平台的说明中说有18个训练集，但下载链接实际上有19个训练集？
3. 背景说明的问题。背景文档只说参照第一阶段大作业，但实际上，第一阶段的文档不足以满足第二阶段的要求。比如说，文档中并没有提到暗噪声的来源问题（是否是其他粒子的事件，或者只是一个随机的触发？），也没有提到契伦科夫辐射的来源问题（只在线源一章中提到过这个名词，但本题是正电子并非线源）。能否考虑对第二阶段大作业做一个额外的说明？
辛苦助教和老师了。

**Comment by Berrysoft** [comment, marked as answer]
1. 我看问的人不多，就忘了发公告。
2. 2到19是18个数吧？
3. 暗噪声的来源我在课上讲了，如果不记得可以去看视频（暗噪声的来源是热效应）。切伦科夫来源在课上说了。至于你的问题，正电子是个点源和它会有切伦科夫辐射不冲突。毕竟我们的探测器的位置分辨率并没有那么好。

**Reply by Berrysoft** [reply]
有关PEType和切伦科夫辐射的事情我会简单发一个公告。切伦科夫辐射的来源大家也都能理解，真的有必要额外说明吗？

**Reply by heroxbd** [reply]
公告已发

**Reply by mhliu0001** [reply]
dbq，是我不会数数

**Reply by MeAPikachu** [reply]
没有final-1文件))
